src/subcommand/wallet/transaction_builder.rs

```diff
@@ -468,6 +476,19 @@ impl TransactionBuilder {
       }
     }
 
+    for input in &transaction.input {
```

Comment:
Should check if marked ordinal winds up in output owned by the user, if so then it's okay.